### PR TITLE
silencing the SDK to ensure it does not panic

### PR DESCRIPTION
**Pull request by damienjburks**
Addresses issue #4303

### CODE REVIEW

File: sdk/options.go [hunk 1]
```diff
@@ -61,12 +61,18 @@ func (o *Options) init() error {
 		o.ConsoleLogger = l
 	}
 
-	bs, err := ioutil.ReadAll(o.Config)
-	if err != nil {
-		return err
+	var bs = []byte("{}")
```
Comment (srenatus) [inline sdk/options.go:64]: [nit] This should be the same as
```suggestion
	bs := []byte("{}")
```

where I think the `bs :=` variant is a bit more consistent with the overall code style

**Review by srenatus — COMMENTED** [review]
🎉 Thanks for picking this up! Would you mind adding a test to assert that the empty config works OK now where it panicked before?

File: sdk/options.go [hunk 2]
```diff
@@ -61,12 +61,18 @@ func (o *Options) init() error {
...
+
+	if o.Config == nil {
+		o.config = bs
```
Comment (johanfylling) [inline sdk/options.go:67]: Also a nit: I think we could inline the value of `bs` here, as there is really no need for it in the outer scope
```suggestion
		o.config = []byte("{}")
```

File: sdk/options.go (hunk 1 above)
Comment (damienjburks) [reply sdk/options.go]: Made the recommended adjustments and added a test case.

File: sdk/options.go (hunk 2 above)
Comment (damienjburks) [reply sdk/options.go:67]: I actually like this one better. Making that adjustment now

File: sdk/opa_test.go [hunk 3]
```diff
@@ -84,6 +84,18 @@ func TestPlugins(t *testing.T) {
...
+func TestPluginPanic(t *testing.T) {
+	ctx := context.Background()
+
+	opa, err := sdk.New(ctx, sdk.Options{})
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer opa.Stop(ctx)
```
Comment (srenatus) [inline sdk/opa_test.go:96]: [nit] there's nothing to gain from a `defer` in this position. Let's just call
```suggestion
	opa.Stop(ctx)
```

**Review by srenatus — APPROVED** [review]
LGTM, one final nitpick

**Comment by srenatus** [comment]
If you wouldn't mind, could you please also squash the commits? A short message including
```
Fixes #4303
```

would be great, too.

File: sdk/opa_test.go (hunk 3 above)
Comment (damienjburks) [reply sdk/opa_test.go]: Added that suggestion.

**Comment by damienjburks** [comment]
@srenatus Everything looks good to me. Please check and confirm when you get a chance.

**Review by srenatus — APPROVED** [review]
LGTM. Thanks!
